salt/metalk8s/products/mounted.sls

```diff
@@ -0,0 +1,65 @@
+{% import_yaml "/etc/metalk8s/bootstrap.yaml" as bootstrap_config %}
```

Comment:
Not really like this approach, strongly prefer using ext_pillar informations, anyone against that ?

Reply:
Fixed